util/kubectl_test.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+import "testing"
+
+func TestCheckKubectl(t *testing.T) {
+	result := CheckKubectl()
```

Comment:
This test seems a bit strange to me as looks like its testing whether kubectl is installed. There could also be problems running it on Circle CI.